starboard/content/ssl/BUILD.gn

```diff
@@ -22,3 +22,11 @@ copy("copy_ssl_certificates") {
   outputs =
       [ "$sb_static_contents_output_data_dir/ssl/certs/{{source_file_part}}" ]
 }
+
+copy("copy_ssl_certificates_evergreen") {
+  if (enable_install_targets) {
```

Comment:
You can remove this entirely

Reply:
This was needed to put the certificates in a `app/cobalt/content` subfolder, which is what [this test assertion](https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/main:starboard/nplb/nplb_evergreen_compat_tests/crashpad_config_test.cc;l=49-52;drc=4a9327f997527c44229030939b296c6a2fa3e120) is looking for.

IOW `copy_ssl_certificates` puts the certs in `out/...//ssl/certs/` and we need them under `out/.../app/cobalt/content/ssl/certs/`.

Reply:
I was saying you can remove the if statement entirely (L27-29)